### PR TITLE
chore(sdk): Improve printed output in wini

### PR DESCRIPTION
**Pull request by timoffex**
Description
-----------
Next step in developing `wini`, improving the printed output using the `rich` library. `rich`

Testing
-------

```sh
rm -rf ./core/dist

# Shows red error message, since package wasn't built yet.
./wini package install  

# Shows commands that are being run.
./wini package release --install
```

### CODE REVIEW

File: tools/wini/subprocess.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import subprocess
+from typing import Mapping
+
+from . import print
+
+
+def check_call(
+    cmd: list[str],
+    *,
+    cwd: str | None = None,
```
Comment (kptkin) [inline tools/wini/subprocess.py:12]: this won't work with earlier versions of python

File: tools/wini/subprocess.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    env: Mapping[str, str] | None = None,
```
Comment (kptkin) [inline tools/wini/subprocess.py:13]: same as above... i think it was added in py3.10.
i think you can use `from __future__ import annotations`

**Review by kptkin — APPROVED** [review]

File: tox.ini [hunk 3]
```diff
@@ -108,6 +108,8 @@ deps=
     wheel
     build
     nox
+    click
+    rich
```
Comment (kptkin) [inline tox.ini:112]: total nit: but maybe we can just use directly our requirement file?

File: tools/wini/subprocess.py (hunk 1 above)
Comment (timoffex) [reply tools/wini/subprocess.py]: fixed

File: tools/wini/subprocess.py (hunk 2 above)
Comment (timoffex) [reply tools/wini/subprocess.py]: fixed

File: tox.ini (hunk 3 above)
Comment (timoffex) [reply tox.ini]: good point, updated! i added requirements_tools to the base deps

**Pull request merged**